base_station/gui/src/components/LCMEcho.vue

```diff
@@ -142,7 +141,9 @@
           {'topic': '/tennis_ball', 'type': 'TennisBall'},
           {'topic': '/imu', 'type': 'IMU'},
           {'topic': '/gps', 'type': 'GPS'},
-          {'topic': '/microscope', 'type': 'Microscope'}
+          {'topic': '/microscope', 'type': 'Microscope'},
+          {'topic': '/zed_gimbal_command', 'type': 'ZedGimbalPosition'},
```

Comment:
Same changes as above, the channel should be zed_gimbal_cmd, & the struct should be ZedGimbalCmd